recce/server.py

```diff
@@ -124,21 +125,44 @@ async def get_lineage(base: Optional[bool] = False):
 @app.post("/api/export", response_class=PlainTextResponse, status_code=200)
 async def export_handler():
     try:
-        return pydantic_model_json_dump(default_recce_state())
+        ctx = default_dbt_context()
+        base = ctx.get_lineage(base=True)
+        current = ctx.get_lineage(base=False)
+        lineage = Lineage(
+            base=base,
+            current=current,
+        )
+        LineageDAO().set(lineage)
+
+        return default_recce_state().export()
     except RecceException as e:
         raise HTTPException(status_code=400, detail=e.message)
 
 
-@app.post("/api/load", status_code=200)
-async def load_handler(file: UploadFile):
+@app.post("/api/import", status_code=200)
+async def import_handler(file: UploadFile):
     try:
         content = await file.read()
-        load_state = RecceState().model_validate_json(content)
+        import_state = RecceState().model_validate_json(content)
         current_state = default_recce_state()
-        current_state.checks = load_state.checks
-        current_state.runs = load_state.runs
+        current_check_ids = [str(c.check_id) for c in current_state.checks]
+        current_run_ids = [str(r.run_id) for r in current_state.runs]
```

Comment:
Could we put the logic in class `RecceState.import()`?

Reply:
okay!